pkg/quota/api/types.go

```diff
@@ -54,6 +54,30 @@ type ClusterResourceQuotaList struct {
 	Items []ClusterResourceQuota
 }
 
+// AppliedClusterResourceQuota mirrors ClusterResourceQuota at a namespace scope, for projection
+// into a namespace
+type AppliedClusterResourceQuota struct {
+	unversioned.TypeMeta
+	// Standard object's metadata.
+	kapi.ObjectMeta
+
+	// Spec defines the desired quota
+	Spec ClusterResourceQuotaSpec
```

Comment:
I'll leave it here as a food for thought, how about just having the Status if we want to show the current state? I am aware it's easier to map entire object, how it's done right now.